karmada-CLI-installtion-and-deploy-example/finish.md

```diff
@@ -0,0 +1,3 @@
+**Summary**
+
+In this scenario, we learned how to initialize the Karmada control plane, join a cluster, and deploy workloads across multiple clusters.
```

Comment:
> and deploy workloads across multiple clusters

Seems we didn't have the step for deploying workloads, right?

Reply:
We plan to work on it separately, and it should be completed in the next PR.